src/app/pages/my-request-detail/my-request-detail.component.ts

```diff
@@ -537,18 +535,6 @@ export class MyRequestDetailComponent implements OnInit, OnDestroy {
     );
   }
 
-  private subscribeDataRequestChanges() {
```

Comment:
This code was added in #281 which solves the issue #255. I've retested this, and it seems by removing this, we have reintroduced this bug, so we need to keep this code here.

Reply:
I have restored this code